app/javascript/components/widgets/weather/hourly-temps.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import gql from 'graphql-tag';
-import { compose, slice, pathOr } from 'ramda';
+import { compose, pathOr, slice } from 'ramda';
```

Comment:
Is there any reason to reorder these here from the previous commit? Does the linter require them to be alphabetical?

Reply:
could we remove slice from the last commit and add it as a new import from ramda in this commit? it's added in the previous commit but unused. it is then used in this commit so it should probably only be introduced here instead of the previous commit.

Reply:
Prefer alphabetical. I don't think we have the rule enforcing it. `slice` was a mistake in the first commit.